<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisconnectTicket</class>
 <widget class="QMainWindow" name="DisconnectTicket">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>936</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonar Disconnect Ticket Creation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QGroupBox" name="Ticket_Buttons">
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Generate Ticket</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Copy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>810</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-21</y>
         <width>351</width>
         <height>829</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="ticketInfo">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Ticket Information:           &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_cstname">
          <property name="text">
           <string>Customer Name:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLineEdit" name="cstName"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_cty">
          <property name="text">
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLineEdit" name="cstCty"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_zone">
          <property name="text">
           <string>Zone:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QComboBox" name="tktZone">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="14" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_contract">
          <property name="text">
           <string>Contract Information:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0" colspan="2">
         <widget class="QLineEdit" name="tktContract"/>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>Account Status:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLineEdit" name="tktStatus"/>
        </item>
        <item row="26" column="0">
         <widget class="QLabel" name="label_datedeactivate">
          <property name="text">
           <string>Date Radio Deactivated</string>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QCalendarWidget" name="tktCalendar">
          <property name="gridVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QDateEdit" name="tktDateText"/>
        </item>
        <item row="29" column="0">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="41" column="0">
         <widget class="QLabel" name="label_notes">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item row="42" column="0">
         <widget class="QTextEdit" name="tktNotes">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Enter any helpful information that may be important, here.</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="31" column="0">
         <widget class="QLineEdit" name="tktSupply"/>
        </item>
        <item row="30" column="0">
         <widget class="QLabel" name="label_pwrsupply">
          <property name="text">
           <string>Power Supply:</string>
          </property>
         </widget>
        </item>
        <item row="33" column="0">
         <widget class="QLineEdit" name="tktRouter"/>
        </item>
        <item row="32" column="0">
         <widget class="QLabel" name="label_mnged">
          <property name="text">
           <string>Managed Router:</string>
          </property>
         </widget>
        </item>
        <item row="34" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>ticketInfo</zorder>
       <zorder>tktCalendar</zorder>
       <zorder>label_datedeactivate</zorder>
       <zorder>tktDateText</zorder>
       <zorder>label_notes</zorder>
       <zorder>tktNotes</zorder>
       <zorder>verticalSpacer_7</zorder>
       <zorder>label_cstname</zorder>
       <zorder>cstName</zorder>
       <zorder>cstCty</zorder>
       <zorder>label_cty</zorder>
       <zorder>label_zone</zorder>
       <zorder>tktZone</zorder>
       <zorder>verticalSpacer_6</zorder>
       <zorder>tktContract</zorder>
       <zorder>label_contract</zorder>
       <zorder>label_status</zorder>
       <zorder>tktStatus</zorder>
       <zorder>verticalSpacer_2</zorder>
       <zorder>tktSupply</zorder>
       <zorder>label_pwrsupply</zorder>
       <zorder>tktRouter</zorder>
       <zorder>label_mnged</zorder>
       <zorder>verticalSpacer_3</zorder>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="ticketTemplate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>tktCalendar</tabstop>
  <tabstop>tktDateText</tabstop>
  <tabstop>tktNotes</tabstop>
  <tabstop>ticketTemplate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tktCalendar</sender>
   <signal>clicked(QDate)</signal>
   <receiver>tktDateText</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>591</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tktDateText</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>tktCalendar</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
